src/views/Imprint.jsx

```diff
@@ -0,0 +1,100 @@
+          <br />
+          <p>
+            <strong>Haftung für Inhalte</strong>
+            <br />
+            <br />
+            Als Diensteanbieter*innen sind wir gemäß § 7 Abs.1 TMG für eigene Inhalte auf diesen Seiten nach den
+            allgemeinen Gesetzen verantwortlich. Nach §§ 8 bis 10 TMG sind wir als Diensteanbieter*innen jedoch nicht
+            verpflichtet, übermittelte oder gespeicherte fremde Informationen zu überwachen oder nach Umständen zu
+            forschen, die auf eine rechtswidrige Tätigkeit hinweisen.
+            <br />
+            <br />
+            Verpflichtungen zur Entfernung oder Sperrung der Nutzung von Informationen nach den allgemeinen Gesetzen
+            bleiben hiervon unberührt. Eine diesbezügliche Haftung ist jedoch erst ab dem Zeitpunkt der Kenntnis einer
+            konkreten Rechtsverletzung möglich. Bei Bekanntwerden von entsprechenden Rechtsverletzungen werden wir diese
+            Inhalte umgehend entfernen.
+            <br />
+            <br />
+            <strong>Haftung für Links</strong>
+            <br />
+            <br />
+            Unser Angebot enthält Links zu externen Websites Dritter, auf deren Inhalte wir keinen Einfluss haben.
+            Deshalb können wir für diese fremden Inhalte auch keine Gewähr übernehmen. Für die Inhalte der verlinkten
+            Seiten sind stets die jeweilige Anbieter*innen oder Betreiber*innen der Seiten verantwortlich. Die
+            verlinkten Seiten wurden zum Zeitpunkt der Verlinkung auf mögliche Rechtsverstöße überprüft. Rechtswidrige
+            Inhalte waren zum Zeitpunkt der Verlinkung nicht erkennbar.
+            <br />
+            <br />
+            Eine permanente inhaltliche Kontrolle der verlinkten Seiten ist jedoch ohne konkrete Anhaltspunkte einer
+            Rechtsverletzung nicht zumutbar. Bei Bekanntwerden von Rechtsverletzungen werden wir derartige Links
+            umgehend entfernen.
+            <br />
+            <br />
+            <strong>Urheberrecht</strong>
+            <br />
+            <br />
+            Die durch die Seitenbetreiber*innen erstellten Inhalte und Werke auf diesen Seiten unterliegen dem deutschen
+            Urheberrecht. Die Vervielfältigung, Bearbeitung, Verbreitung und jede Art der Verwertung außerhalb der
+            Grenzen des Urheberrechtes bedürfen der schriftlichen Zustimmung der jeweiligen Autor*innen bzw.
+            Ersteller*innen. Downloads und Kopien dieser Seite sind nur für den privaten, nicht kommerziellen Gebrauch
+            gestattet.
+            <br />
+            <br />
+            Als Betreiber*innen möchten wir darauf hinweisen, dass der Quellcode dieser Website als Open Source Projekt
+            unter folgender Adresse veröffentlicht wurde:&nbsp;
+            <Link to="https://github.com/kenodressel/quarantine-hero">
+              https://github.com/kenodressel/quarantine-hero
+            </Link>
+            <br />
+            <br />
```

Comment:
As the anchor tag seams to already introduce a line break, removing one break here will keep the distance between paragraphs consistent.
```suggestion
            <br />
```